src/components/Combobox/Combobox.tsx

```diff
@@ -80,6 +83,7 @@ function Combobox<Item>({
   onChange,
   value,
   items,
+  variant = 'outline',
```

Comment:
This design is the one we already display right? i am asking so dont have to go update everything!

Reply:
From what i am seeing that's a case, just trying to confim it.

Reply:
Yeap, that's intended! Thanks for the comment.